doc/userdoc/installation/cmake_options.rst

```diff
@@ -28,70 +28,116 @@ Options for configuring NEST
++---------------------------------------------+----------------------------------------------------------------+
+
+**Change NEST behavior:**
+
++--------------------------+----------------------------------------------------+
+| -Dtics_per_ms=[number]   |  Specify elementary unit of time. [default 1000.0] |
++--------------------------+----------------------------------------------------+
+| -Dtics_per_step=[number] |  Specify resolution. [default 100]                 |
++--------------------------+----------------------------------------------------+
+
+**Add user modules:**
+
++---------------------------------------------+----------------------------------------------------------------+
+| -Dexternal-modules=[OFF|<list;of;modules>]  | External NEST modules to be linked in, separated by ';'.       |
+|                                             | [default=OFF]                                                  |
++---------------------------------------------+----------------------------------------------------------------+
+
+**Connect NEST with external projects:**
+
++---------------------------------------------+----------------------------------------------------------------+
+| -Dwith-libneurosim=                         | Request the use of libneurosim. Optionally give the directory, |
+| [OFF|ON|</path/to/libneurosim>]             | where libneurosim  is installed. [default=OFF]                 |
++---------------------------------------------+----------------------------------------------------------------+
+| -Dwith-music=[OFF|ON|</path/to/music>]      | Request the use of MUSIC. Optionally give the directory, where |
+|                                             | MUSIC  is installed. [default=OFF]                             |
++---------------------------------------------+----------------------------------------------------------------+
+| -Dwith-sionlib=[OFF|ON|</path/to/sionlib>]  | Request the use of sionlib. Optionally give the directory,     |
+|                                             | where sionlib  is installed. [default=OFF]                     |
++---------------------------------------------+----------------------------------------------------------------+
+
+**Cross-compiling:**
+
+Should be set via toolchain files
+
++---------------------------------------------+----------------------------------------------------------------+
+| -Denable-bluegene=[OFF|STRING]              | Configure for BlueGene. [default=OFF]                          |
++---------------------------------------------+----------------------------------------------------------------+
+| -Dk-computer=[OFF|ON]                       | Enable K computer. [default=OFF]                               |
++---------------------------------------------+----------------------------------------------------------------+
+| -Dtarget-bits-split=["standard"|STRING]     | Split of the 64-bit target neuron identifier type. "standard"  |
```

Comment:
Combine this with detailed timers and time steps into one section. And shouldn't it be `...=["standard"|"hpc"]`?